.gitignore

```diff
@@ -21,8 +21,9 @@
 .DS_Store
 *.swp
 
-# binary
```

Comment:
FIX: We should still keep this in the gitignore because someone somewhere is going to run `go install` and the accidentally commit this. 🤷‍♀️